Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/system/classes/EditClassLayout.java

```diff
@@ -131,14 +133,14 @@ public void execute(Boolean value) {
      *
      * @param name Class name
      * @param groups Class groups
-     * @param engine Class engine
+     * @param engines Class engines
      */
-    public void setClass(String name, String groups, String engine) {
+    public void setClass(String name, String groups, String engines) {
 
         if (name != null) {
             this.nameField.setValue(name);
             this.nameField.setDisabled(true);
-            this.enginesPickList.setValue(engine);
+            this.enginesPickList.setValues(engines.split(", "));
```

Comment:
Actually, since split() returns a String[], we can't simply apply a trim() to it. 
We could remove empty spaces like this : split("\\s*,\\s*"). 
Would that be fine with you ?